### PR TITLE
Add basic citus upgrade tests

### PR DESCRIPTION
**Pull request by SaitTalhaNisanci**
This PR adds simple upgrade tests, it checks:
- Upgrading citus version is successful with server restart
- Upgrading citus version without restarting server forces you to restart server

To run the test run :
```
fab run.update
```

### CODE REVIEW

File: fabfile/constants.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DROP_CITUS = 'DROP EXTENSION IF EXISTS citus;'
+CREATE_CITUS = 'CREATE EXTENSION citus;'
+ALTER_CITUS = 'ALTER EXTENSION citus UPDATE;'
```
Comment (hanefi) [inline fabfile/constants.py]: If we do not provide `TO version` at the end, this will update to the latest version available. Is this really what we want?

Comment (hanefi) [reply fabfile/constants.py]: and maybe have aline ending at the end as well

File: fabfile/run.py [hunk 2]
```diff
@@ -198,3 +199,42 @@ def tpch_queries(query_info, connectionURI, pg_version, citus_version):
         out_val = run(run_string)
         results_file.write(out_val)
         results_file.write('\n')
+
+@task
+@runs_once
+@roles('master')
+def update():
+     citus_versions = ['release-8.0', 'release-8.1']
+     pg_version = '11.5'
```
Comment (hanefi) [inline fabfile/run.py]: similarly, instead of a hardcoded value, can this be a parameter as well?

Comment (hanefi) [reply fabfile/run.py]: An alternative solution is to use `config.settings['pg-version']`
This way we can override this by adding some initial use commands. For example

	fab use.postgres:12.0 run.update:8.1:8.2

**Review by hanefi — CHANGES_REQUESTED** [review]
I did not test the changes here myself yet. Left some initial reviews. Looks good overall

**Comment by SaitTalhaNisanci** [comment]
@hanefi we know have citus upgrade tests in our CI, I think we can close this, I dont see any benefit of this PR now, what do you think?

**Comment by SaitTalhaNisanci** [comment]
Closing this since we have these tests as part of our CI jobs now.

**Pull request closed without merging**